be/src/service/doris_main.cpp

```diff
@@ -153,6 +154,7 @@ int main(int argc, char** argv) {
     // options
     doris::EngineOptions options;
     options.store_paths = paths;
+    options.backend_uid = doris::UniqueId();
```

Comment:
Is `backend_uid` act like a random seed that will be generated every time BE started?
If so, I think the name is confused that make people think of it as the ID of a BE?
Maybe `random_seed` is better?